src/mrpro/operators/_MatrixOp.py

```diff
@@ -0,0 +1,56 @@
+"""Matrix Multiplication Operator."""
+
+# Copyright 2024 Physikalisch-Technische Bundesanstalt
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#       http://www.apache.org/licenses/LICENSE-2.0
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+
+import torch
+
+from mrpro.operators import LinearOperator
+
+
+class MatrixOp(LinearOperator):
```

Comment:
This would also require a test

Reply:
I am not sure whether a test is really necessary here, since this is really not more than a wrapper for torch's @ operator. What do we expect could go wrong?

Reply:
> What do we expect could go wrong?

famous last words. 

In general our tests should cover as much of our code as possible. We might change some interfaces of the Operators and then we might need to adapt this interface too. If there is no test it is very likely that we would miss this. Nevertheless, probably @fzimmermann89 PR will add a test for this.